recipes/openblas/all/conanfile.py

```diff
@@ -38,13 +86,15 @@ class OpenblasConan(ConanFile):
         "use_thread": True,
         "use_locking": True,
         "dynamic_arch": False,
+        "target": None,
     }
     options_description = {
         "build_lapack": "Build LAPACK and LAPACKE",
         "build_relapack": "Build with ReLAPACK (recursive implementation of several LAPACK functions on top of standard LAPACK)",
         "use_thread": "Enable threads support",
         "use_locking": "Use locks even in single-threaded builds to make them callable from multiple threads",
         "dynamic_arch": "Include support for multiple CPU targets, with automatic selection at runtime (x86/x86_64, aarch64 or ppc only)",
+        "target": "OpenBLAS TARGET variable (see TargetList.txt)",
```

Comment:
Where is TargetList.txt?

Reply:
It's part of OpenBLAS itself: https://github.com/OpenMathLib/OpenBLAS/blob/4e9144b39faa6b0542e1b7c183e0dab1a8637a07/TargetList.txt